lib/Sema/ConstraintSystem.cpp

```diff
@@ -3516,10 +3516,26 @@ void ConstraintSystem::resolveOverload(ConstraintLocator *locator,
   if (isDebugMode()) {
     PrintOptions PO;
     PO.PrintTypesForDebugging = true;
-    llvm::errs().indent(solverState ? solverState->depth * 2 : 2)
-      << "(overload set choice binding "
-      << boundType->getString(PO) << " := "
-      << adjustedRefType->getString(PO) << ")\n";
+
+    if (!OpenedTypes.empty()) {
+      auto &log = llvm::errs();
+      log.indent(solverState ? solverState->depth * 2 : 2);
```

Comment:
Let's land https://github.com/apple/swift/pull/60443/commits/5254d00abc861de8dc007180755fa4e3118c314e as a separate PR and update this code to use `getCurrentIndent()`.